package-lock.json

```diff
@@ -20072,7 +20072,7 @@
             "ms": {
               "version": "2.0.0",
               "resolved": "https://registry.npmjs.org/ms/-/ms-2.0.0.tgz",
-              "integrity": "sha512-Tpp60P6IUJDTuOq/5Z8cdskzJujfwqfOTkrwIwj7IRISpnkJnT6SyJ4PCPnGMoFjC9ddhal5KVIYtAt97ix05A=="
+              "integrity": "sha1-VgiurfwAvmwpAd9fmGF4jeDVl8g="
```

Comment:
Weaker sha alert!

Reply:
oops, I committed a bunch of files including localization stuff, reverting